api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -55,6 +55,18 @@ type PodProbeSpec struct {
 	PeriodSeconds       int `json:"periodSeconds"`
 }
 
+// ConfigMapNameReference references a config map in a specific namespace.
+// The namespace must be specified at the point of use.
+type ConfigMapNameReference struct {
+	Name string `json:"name,omitempty"`
+}
+
+// DriverRepoConfigSpec defines custom repo configuration for OFED container
+type DriverRepoConfigSpec struct {
```

Comment:
not being used anymore, can you remove ?

Reply:
done